repository/kafka/docs/v0.2/versions.md

```diff
@@ -0,0 +1,5 @@
+### Versions
+
+- Apache Kafka 2.12-2.3.0
+- OpenJDK 8
+- JMX Exporter 0.1.0
```

Comment:
As mentioned in the other PR, I think it's important to mention the compatible KUDO version here as installing the wrong one might cause issues with the Kafka operator.

Reply:
something like this: 

### KUDO Version Requirement

KUDO version 0.4.0 or later